salt/metalk8s/volume/files/loopdevice.service.j2

```diff
@@ -0,0 +1,17 @@
+{# source salt linux formulars #}
+[Unit]
+Description=Setup loop devices for {{ device_name }}
+DefaultDependencies=no
+After=systemd-udev-settle.service
+Before=local-fs.target
+Wants=systemd-udev-settle.service
+
+[Service]
+Type=oneshot
+{# The command is prefixed with '-' to consider it a success if the loopback device is already setup #}
+ExecStart=-/sbin/losetup {{ device_name }} {{ file }}
```

Comment:
How would you figure out a reasonable `device_name`?

Reply:
I think we are almost certain that having something like 
`/sbin/losetup/ <volume_uid> <sparsefile_name>` could work no?? Since it is impossible for `device_name` with same volume_uid or volume_name to exist.

For `device_name` I probably will use the unique `volume_uid`

Reply:
Hmmm, seems we will be having multiple loopdevice for one particular volume :(